modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.scss

```diff
@@ -43,6 +43,11 @@
 		}
 	}
 
+	&-title {
```

Comment:
@rebsilva I think this changes are related to the previous commit, right? Next time, it's better to put them together in the same commit.

Reply:
Actually is related to this commit, related to this part:

`lfr__object-web-view-object-definitions-folder-list-title`

It's only because I've put the css code inside the ViewObjectDefinitions.scss file since it is the "global" one you know? If you think I could change this please let me know!

Reply:
You are right! I think I was confused since this has a title class too, thanks for letting me know! purple_heart